FlowCrypt/Functionality/Mail Provider/Message Provider/MessageService.swift

```diff
@@ -230,7 +230,8 @@ extension MessageService {
         let pubKeys = try localContactsProvider.retrievePubKeys(for: sender.email, shouldUpdateLastUsed: false)
         if pubKeys.isNotEmpty || onlyLocal { return pubKeys }
 
-        guard let contact = try? await pubLookup.fetchRemoteUpdateLocal(with: sender)
+        try await pubLookup.fetchRemoteUpdateLocal(with: sender)
```

Comment:
should use `try?` - I think this was intentional to ignore errors, since local pubkeys can be used most of the time, if present. Would be good to add a comment to explain this reason

Reply:
ok